README.md

```diff
@@ -57,11 +57,24 @@ The entire API is `get` and `set`, and takes a range of inputs:
 >>> await plc.set('y101', True)  # Sets Y101 to true
 ```
 
+If a path is provided to a tags file when the driver is initialized, 
```

Comment:
I'd add short commentary on what a tags file is and how to download it from the PLC software. Doesn't need to be detailed or have pictures, but enough to make this clear. Maybe add a subheader like "Using Tags"

Reply:
👍